alpine/docker-entrypoint.sh

```diff
@@ -27,7 +22,7 @@ if [[ "$1" == "kong" ]]; then
       setcap cap_net_raw=+ep /usr/local/openresty/nginx/sbin/nginx
     fi
 
-    exec su-exec kong /usr/local/openresty/nginx/sbin/nginx \
+    /usr/local/openresty/nginx/sbin/nginx \
```

Comment:
still need the `exec` or you get a bash process hanging around.

Reply:
I can tell you this container has been working fine for me (also for my Enterprise deployment) using docker-compose, Kubernetes and OpenShift. I do not have access to a full test environment so but use all the functions normally